app/views/products/index.html.erb

```diff
@@ -0,0 +1,26 @@
+<section class="all-products">
+  <span class="index-banner">
+    <% if @products.count == 0 %>
+    Yikes! No products meet those parameters.
+    <% else %>
+    Product Gallery
+    <% end %>
+  </span>
+
+  <% @products.each do |product| %>
+  <section class="info-card">
+    <section class="product">
+      <% if product.photo.file? %>
+      <span class= "index-image"><%= link_to image_tag(product.photo.url), product_path(product.id) %></span>
+      <% else %>
+      <span class= "index-image"><%= link_to image_tag("http://www.visitbuffaloniagara.com/content/uploads/sv-event-images/434cfb09-bb34-449b-ac6c-d400b6a53f1d.jpg"), product_path(product.id), alt: "A product image placeholder" %></span>
+      <% end %>
+      <span><em><%= link_to product.name.upcase, product_path(product.id) %></em></span>
+      <span><%= product.price %></span>
```

Comment:
Your `product.price` needs to be formatted to a correct dollar format. Otherwise numbers appear like `100.0` instead of `100.00`.

The other locations for this that I could find are:
- Product detail page
- Every Merchant order page